officers/worf.json

```diff
@@ -0,0 +1,89 @@
+{
+  "description": "Fearless warrior. A man of two worlds.\nWorf, son of Mogh, is a Klingon waarior and Starfleet officer. At a young age, Worf lived on the Khitomer colony, which was eventually attacked by Romulan forces, leaving him orphaned. Luckily, he was rescued by a Federation vessel, the USS Intrepid, where he was adopted by the Rozhenko familu. Worf grew up surrounded by humans, but kept close to his ancestral home. As the first of his kind to ever serve in Starfleet, he was constantly faced ridicule and rejection from fellow Klingons. Worf fights twice as hard for his honor, and has become a battle-sharpened edge as a result. Worf's place as Chief Security aboard the Enterprise came after a stint as conn officer. He regularly reports to away team duty as the fron line of defense for the landing party. Worf is a fearless and formidable warrior, with a bat'leth, or helming the tactical controls of a starship.",
+  "ability_info": {
+    "captain_maneuver": {
+      "name": "Headlong into battle",
+      "bonus": "{value} to Warp Speed after winning a battle (not cumulative)",
+      "percentage": true,
+      "value": 2
+    },
+    "officer_ability": {
+      "name": "Only Fools Have No Fear",
+      "bonus": "{value} to all Mitigation against non-player targets",
+      "percentage": true
+    }
+  },
+  "class": "command",
+  "faction": "federation",
+  "group": "tng crew",
+  "img_url": "./assets/worf.png",
+  "officer_name": "Worf",
+  "ranks": [
+    {
+      "name": "Ensign I",
+      "officer_ability_value": 0.2,
+      "rank_id": 1,
+      "requirements": {
+        "badges": 0,
+        "credits": 0,
+        "others": [],
+        "shards": 38,
+        "xp": 0
+      }
+    },
+    {
+      "name": "Lieutenant JG II",
+      "officer_ability_value": 0.3,
+      "rank_id": 2,
+      "requirements": {
+        "badges": 0,
+        "credits": 0,
+        "others": [],
+        "shards": 55,
+        "xp": 10000
+      }
+    },
+    {
+      "name": "Lieutenant III",
+      "officer_ability_value": 0.4,
+      "rank_id": 3,
+      "requirements": {
+        "badges": 0,
+        "credits": 1200,
+        "others": [],
+        "shards": 114,
+        "xp": 100000
+      }
+    },
+    {
+      "name": "Lt. Commander IV",
+      "officer_ability_value": 0.5,
+      "rank_id": 4,
+      "requirements": {
+        "badges": 1,
+        "credits": 4800,
+        "others": [],
+        "shards": 156,
+        "xp": 250000
+      }
+    },
+    {
+      "name": "Commander V",
+      "officer_ability_value": 0.6,
+      "rank_id": 5,
+      "requirements": {
+        "badges": 2,
+        "credits": 12000,
+        "others": [],
+        "shards": 225,
+        "xp": 500000
+      }
+    }
+  ],
+  "rarity": "rare",
+  "synergy": {
+    "command": 1,
```

Comment:
I think those might be wrong. I think it's 0.1 for command and 0.15 for science and engineering

Reply:
I actually couldn't work out the synergies 😅